Src/CSharp/Net Core/TDD-Katas-NETCore.csproj

```diff
@@ -10,8 +10,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0-preview-20180320-02" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.6.1" />
     <PackageReference Include="NUnit" Version="3.10.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.9.0" />
```

Comment:
You may as well update to the 3.10 release of the adapter.

Reply:
Thanks @rprouse for looking into it